.github/workflows/docker-hardfork-ci.yml

```diff
@@ -45,12 +38,13 @@ jobs:
         -t "scorum/${NET_NAME}:0.${HF_VERSION}.0.${GIT_COMMIT}" .
 
     - name: Push docker image to registry
+      if: github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/master'
```

Comment:
why do you use master branch in build-testnet job?

Reply:
okay, lets build testnet image only from develop branch.